build.py

```diff
@@ -16,3 +16,4 @@
 @init
 def set_properties(project):
     project.build_depends_on("mockito")
+    project.set_property("coverage_threshold_warn", 90)
```

Comment:
Can we also make `coverage_branch_threshold_warn` and `coverage_branch_partial_threshold_warn` to 80%.

Reply:
Sure